src/stylesheets/elements/_inputs.scss

```diff
@@ -263,76 +263,90 @@ legend {
 
 // Range inputs
 
-[type=range] {
-  appearance: none;
-  border: none;
-  padding-left: 0;
-  width: 100%;
-}
-
-[type=range]::-webkit-slider-runnable-track {
-  background: $color-gray-light;
-  border: 1px solid $color-gray-medium;
-  cursor: pointer;
-  height: 1.2rem;
-  width: 100%;
+// Change to $color-focus in 2.0
+@mixin range-focus {
+  box-shadow: 0 0 0 2px $color-primary;
 }
 
-[type=range]::-moz-range-track {
-  background: $color-primary;
+@mixin range-track {
+  background: $color-gray-lighter;
   border: 1px solid $color-gray-medium;
   cursor: pointer;
-  height: 1.2rem;
-  width: 100%;
-}
-
-[type=range]::-ms-track {
-  background: transparent;
-  color: transparent;
-  cursor: pointer;
-  height: 1.2rem;
+  height: 1.6rem;
   width: 100%;
 }
 
-[type=range]::-webkit-slider-thumb {
-  -webkit-appearance: none;
-  border: 1px solid $color-gray-medium;
-  height: 2.2rem;
-  border-radius: 1.5rem;
-  background: $color-gray-lightest;
-  cursor: pointer;
-  margin-top: -.65rem;
-  width: 2.2rem;
-}
-
-[type=range]::-moz-range-thumb {
+@mixin range-thumb {
   background: $color-gray-lightest;
-  border: 1px solid $color-gray-medium;
-  border-radius: 1.5rem;
-  cursor: pointer;
-  height: 2.2rem;
-  width: 2.2rem;
-}
-
-[type=range]::-ms-thumb {
-  background: $color-gray-lightest;
-  border: 1px solid $color-gray-medium;
+  box-shadow: 0 0 0 1px $color-gray-medium;
   border-radius: 1.5rem;
   cursor: pointer;
-  height: 2.2rem;
-  width: 2.2rem;
+  height: 2.5rem;
+  width: 2.5rem;
 }
 
-[type=range]::-ms-fill-lower {
+@mixin range-ms-fill {
   background: $color-gray-light;
   border: 1px solid $color-gray-medium;
   border-radius: 2rem;
 }
 
-[type=range]::-ms-fill-upper {
-  background: $color-gray-light;
-  border: 1px solid $color-gray-medium;
-  border-radius: 2rem;
+[type=range] {
```

Comment:
Did we want to use a class here, instead of targeting the actual HTML element? Or is targeting a specific subset of elements ok?

I don't remember exactly where we landed but I thought that we were trying to move away from tag-based css